### PR TITLE
Web readme update

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Added guide to use wallet connect and MyAlgo wallet for web package

### CODE REVIEW

File: packages/web/README.md [hunk 1]
```diff
@@ -24,9 +24,25 @@ In the `@algo-builder/web` package we pass transaction [parameters](https://gith
...
-    const web = new WebMode(AlgoSigner, CHAIN_NAME);
+1.  ##### AlgoSigner:
+    Create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
+
+        const connector = new WebMode(AlgoSigner, CHAIN_NAME);
+
+2. ##### MyAlgo Wallet:
+    Create an instance of the `MyAlgoWalletSession` class by passing the chain name and connect it using `connectToMyAlgo`.
+
+         const connector = new MyAlgoWalletSession(CHAIN_NAME)
```
Comment (ratik21) [inline packages/web/README.md:37]: how about using `session`/`wcSession` instead of `connector` here?

File: packages/web/README.md [hunk 2]
```diff
@@ -24,9 +24,25 @@ In the `@algo-builder/web` package we pass transaction [parameters](https://gith
 
 ### Example
 
-To use `web` package in your react app, first you need to create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
+You can connect to `web` package in your react app by using different wallets.
```
Comment (ratik21) [inline packages/web/README.md:27]:
```suggestion
You can connect to `web` package in your react app by using different wallets. Currently supported wallets include: 
```

File: packages/web/README.md [hunk 3]
```diff
@@ -24,9 +24,25 @@ In the `@algo-builder/web` package we pass transaction [parameters](https://gith
...
+
+         const connector = new MyAlgoWalletSession(CHAIN_NAME)
+         await connector.connectToMyAlgo();
+
+3. ##### Wallet Connect:
+    Create an instance of the `WallectConnectSession` class by passing the chain name and create a new session using `create` and connect to it using `onConnect`.
+
+        const connector = new WallectConnectSession(CHAIN_NAME);
+        await connector.create(true);
+        connector.onConnect((error, response) => console.log(error, response));
```
Comment (ratik21) [inline packages/web/README.md:46]: Wrap this in ```js ..

File: packages/web/README.md [hunk 4]
```diff
@@ -24,9 +24,25 @@ In the `@algo-builder/web` package we pass transaction [parameters](https://gith
 
 ### Example
 
-To use `web` package in your react app, first you need to create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
+You can connect to `web` package in your react app by using different wallets.
 
-    const web = new WebMode(AlgoSigner, CHAIN_NAME);
+1.  ##### AlgoSigner:
+    Create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
```
Comment (ratik21) [inline packages/web/README.md:30]: I think we should link a template/example where this integration is used in a webapp for each wallet

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Let's update per Ratik suggestions.

**Pull request merged**